router/ui-routes.js

```diff
@@ -108,6 +121,51 @@ router.get("/admin/exec", function(req, res) {
         res.render("admin_exec", {
             whiteBackground: true,
             currentExec: exec,
+            roles: execRoles,
+            initialRole: execRoles[0].name,
+        }, function(err, html) {
+            if (err) {
+                logger.error(err);
+                return res.render("error", {whiteBackground: true, message: err.message, status: 500, err:err, });
+            // eslint-disable-next-line no-else-return
+            } else {
+                res.send(html);
+            }
+        });
+    })
+    .catch((err) => {
+        return res.render("error", {whiteBackground: true, message: err.message, status: 500, });
+    });
+});
+
+/**
+ * Get admin roster page for executives of a certain year
+ * 
+ * @param exec Executive Role to be queried
+ * @param year Year which is to be looked up
+ */
+router.get("/admin/:exec/:year", function (req, res) {
```

Comment:
the UI page should be `/admin/exec/:year` where you can set each of the executives, not  `/admin/:exec/:year` Other then that I think the page looks awesome and I like the functionality of the table to add or edit the execs.

Reply:
I wrote this with the intent in mind to leverage handlebars, and it's ability to serve up static html pages. It re-uses the same exec-admin template (admin_exec.hbs) while being able to dynamically grab information from the database.

Reply:
yup we still want to use handlebars, but we want to be able to show the execs by year rather than by role.

Reply:
change has been made to accept year as a param instead of exec and year